registry/eip155/autonomys.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "autonomys",
+  "shortName": "Autonomys",
+  "fullName": "Autonomys Mainnet",
+  "aliases": ["evm-870", "autonomys-mainnet"],
+  "caip2Id": "eip155:870",
```

Comment:
Looks like it's not in ethereum list yet? 
If you are sure about the future chain id add the exception here to make it pass the check: https://github.com/graphprotocol/networks-registry/blob/3019278a9dcf645e022da33c5fd3c644601b6715/src/validate_logic.ts#L17